resources/views/projects/form.blade.php

```diff
@@ -0,0 +1,52 @@
+<form action="@isset($project) {{ route('projects.update', $project->id) }} @else {{ route('projects.store') }} @endisset" class="form-validate is-alter" id="project_form">
+    @isset($project)
+        @method('PUT')
+    @endisset
+        @csrf
+        <div class="modal-dialog" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title">@isset($project) Edit @else Add @endisset Project</h5>
+                    <a href="#" class="close" data-dismiss="modal" aria-label="Close">
+                        <em class="icon ni ni-cross"></em>
+                    </a>
+                </div>
+                <div class="modal-body" id="employee_form_div">
+            <div class="form-group">
+            <label class="form-label required" for="name">Name</label>
+            <div class="form-control-wrap">
+                <input type="text" class="form-control" value="@isset($project){{ $project->name }}@endisset" name="name" required>
+            </div>
+        </div>
+        <div class="form-group">
+            <label class="form-label required" for="detail">Detail</label>
+            <div class="form-control-wrap">
+                <input type="text" class="form-control" id="detail" value="@isset($project){{ $project->detail }}@endisset" name="detail" required>
```

Comment:
use textarea as detail because detail can be long description

Reply:
Done.